modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying architectures on the Agent-based installer 
+
+You can check which architecture your {product-title} agent installer cluster is using by viewing the version binary.
+
+.Prerequisites: 
+
+* You installed the OpenShift CLI (oc)
+* You have the openshift-installer binary
+
+.Procedure: 
+
+. Check your release payload by running the following command: 
+[source,terminal]
++
+----
+$ ./openshift-install version
+----
++
+.Example output 
+[source,terminal]
+----
+./openshift-install 4.15.0-ec.3
+built from commit e80d50ac8c9ab65833045d648a01eb1a0a3a6d7d
+release image quay.io/openshift-release-dev/ocp-release@sha256:27d41f99e0784a2f596e773799983147b794df48d1f7536d51a95a60945b1931
+release architecture amd64
+----
++
+If you are using the `amd64` or `multi` installer payload, the `release architecture` output of this command displays to `amd64` by default
+
+. To verify that you are using your preferred architecture, run the following command:
+[source,terminal]
++
+----
+$ oc adm release info quay.io/openshift-release-dev/ocp-release@sha256:27d41f99e0784a2f596e773799983147b794df48d1f7536d51a95a60945b1931 -o jsonpath="{ .metadata.metadata}"
+----
++
+.Example output of a cluster using the `multi` binary
+[source,terminal]
+----
+release@sha256:27d41f99e0784a2f596e773799983147b794df48d1f7536d51a95a60945b1931 -o jsonpath="{ .metadata.metadata}"
+{"release.openshift.io/architecture":"multi"}
+----
++
+With the `multi` installer binary, your cluster supports `arm64`, `amd64`, `s390x`, and `ppc64le`. Otherwise, you can only install the architecture defined in the `release architecture` output from the `./openshift-install version` command. 
```

Comment:
We should wait and see if the enablement work for s390x and ppc64le make 4.15, assuming they do this should be fine.  cc: @jaypoulz

Reply:
According to our testing, we're looking good for ppc64le and s390x for 4.15.

Would be good to get QE to put the the next multi payload through it's paces for those arches tho.